flow/Makefile

```diff
@@ -129,7 +129,7 @@ SHELL          := /usr/bin/env bash
 # - the following settings allowed user to point OpenROAD binaries to different
 #   location
 # - default is current install / clone directory
-export FLOW_HOME ?= .
+export FLOW_HOME ?= $(shell pwd)
```

Comment:
Does this cause 'make issue..' to use absolute paths rather than relative ones?  That would be undesirable.

Reply:
Good catch in terms of thinking of edge cases. I didn't think to test it before you mentioned it.

Works fine though:

```
$ make floorplan_issue
$ grep /home  vars-gcd-nangate45-base.sh  |wc -l
0
$ head -n 10 vars-gcd-nangate45-base.sh 
export FLOW_HOME=${FLOW_HOME:-$(pwd)}
export RCX_RULES="${FLOW_HOME}/platforms/nangate45/rcx_patterns.rules"
export DESIGN_NAME="gcd"
export SDC_FILE="./designs/nangate45/gcd/constraint.sdc"
export IO_PLACER_H="metal5"
export GDS_FILES="${FLOW_HOME}/platforms/nangate45/gds/NangateOpenCellLibrary.gds"
export CTS_BUF_CELL="BUF_X4"
export STREAM_SYSTEM="GDS"
export SYNTH_HIERARCHICAL="0"
export LATCH_MAP_FILE="${FLOW_HOME}/platforms/nangate45/cells_latch.v"
```